feeds/package.json

```diff
@@ -32,6 +32,7 @@
     "setup": "tsc -b"
   },
   "dependencies": {
+    "@chainlink/json-api-client": "0.0.1",
```

Comment:
I had considered pulling out `fetchWithTimeout` into a separate package but figured once we have a backend we'd probably want to return JSON-API responses to be consistent across our apps.